src/Protocol/ActivityPub/Processor.php

```diff
@@ -612,6 +612,7 @@ public static function updatePerson($activity)
 
 		Logger::log('Updating profile for ' . $activity['object_id'], Logger::DEBUG);
 		APContact::getByURL($activity['object_id'], true);
+//		Contact::updateFromProbe($activity['object_id'], $network = '', $force = false)
```

Comment:
Do you plan on removing this line at some point?

Reply:
This is WIP